### ISSUE TITLE
Add Linux OS Iso Support

### ISSUE THREAD

**Issue by LievreAI** [pbatard/Fido#47]
can you please add option to download linux such as , ubuntu , centos , kali iso download in the script?

**Comment by pbatard** [comment]
This has been asked before (I think in the Rufus issue tracker rather than here) but I have no plans to do so, because, first, Linux distro maintainers don't make it as obnoxious as Microsoft to get the their ISOs, and second, because I'd have to __permanently__ maintain a list of mirror for each distros, and it would quickly become a massive time sink to maintain.

Plus, there are so many distros that people would be constantly asking for their favourite one to be added (with a _"there's no reason not to add X if you added Y"_ argument).

In short, the cost/benefit ratio of doing so is way too high, and I consider that getting access to a Linux ISO is easy enough for all user not to require Fido to support such downloads.
